include/onnxruntime/core/providers/tensorrt/tensorrt_execution_provider_utils.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <fstream>
+#include <unordered_map>
+#include "flatbuffers/idl.h"
+
+// Deserialize engine profile
+std::unordered_map<std::string, std::unordered_map<size_t, std::pair<int64_t, int64_t>>> DeserializeProfile(std::ifstream& infile) {
```

Comment:
i think it would make sense to move SerializeProfile() to this file as well?

Reply:
Yes, we still need to move SerializeProfile() to here in the end and add additional test for serialization. Right now we don't have such test for serialization.